deploy/branding_config.yaml

```diff
@@ -3,17 +3,31 @@ kind: ConfigMap
 metadata:
   name: branding-config
   namespace: bold-services
+# All the branding images variables are accepted as URL.
+# Image type: png, svg, jpg, jpeg
+# Ex: https://example.com/loginlogo.jpg
 data:
+  # This is header logo for the application and the preferred image size is 40 x 40 pixels.
   BOLD_SERVICES_BRANDING_MAIN_LOGO: ""
   
+  # This is login logo for the application and the preferred image size is 200 x 40 pixels.
   BOLD_SERVICES_BRANDING_LOGIN_LOGO: ""
   
+  # This is email logo, and the preferred image size is 200 x 40 pixels.
```

Comment:
This is an email logo, and the preferred image size is 200 x 40 pixels.

Reply:
concern addressed